src/accounts/accounts.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Model, Table } from 'sequelize-typescript';
+
+@Table
+export class Accounts extends Model<Accounts> {
+  @Column({ primaryKey: true, autoIncrement: true })
+  id: number;
+
+  @Column
```

Comment:
The type property is missing here. Sequelize can't read through typescript types. For sequelize to understand what is the type of the column we need to explicitly pass the type parameter.
@Column({  type: DataType.STRING })